app/assets/javascripts/javascripts/google_analytics_tracker.js

```diff
@@ -0,0 +1,15 @@
+var googleAnalyticsTracker = function(){
+  var init = function(trackerId){
+    _injectTracker("ga");
+    ga("create", trackerId, "auto");
+    ga("send", "pageview");
+  };
+
+  var _injectTracker = function(globalObjectName){
+    (function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js",globalObjectName);
+  };
+
+  return {
+    init: init
+  }
+}();
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.